frappe/desk/doctype/todo/todo.py

```diff
@@ -64,6 +64,10 @@ def delete_communication_links(self):
 			"link_name": self.name
 		})
 
+	def set_owner(self):
+		# owner is set from client side
+		pass
+
```

Comment:
Shouldn't be allowed to change this value after the document's been created though?

---

I think we should just go with this approach -> https://github.com/frappe/frappe/pull/14404#issuecomment-939706548